.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.6
+  - 2.1.10
```

Comment:
try those keys, but I'm not sure https://github.com/rebelidealist/stripe-ruby-mock/blob/updated-stripe-keys/.travis.yml#L16